back/app/controllers/omniauth_callback_controller.rb

```diff
@@ -75,6 +75,7 @@ def auth_callback(verify:, authver_method:)
         handle_verification(auth, @user) if verify
         redirect_to(add_uri_params(Frontend::UrlService.new.signup_success_url(locale: @user.locale), omniauth_params))
       rescue ActiveRecord::RecordInvalid => e
+        Sentry.configure_scope { |scope| scope.set_context('auth object', auth) }
```

Comment:
I'm wondering if you don't need to do something like `auth.to_h` instead of `auth` to have the content being displayed in Sentry, but I could be wrong. Otherwise perhaps you would see something like `#<AuthObject:0x401b3a38>`

Reply:
@sebastienhoorens  Is what I show in the screenshot not sufficient?

Reply:
Oh yes didn't see that, looks perfect! :D